synapse/cortex.py

```diff
@@ -3057,6 +3044,30 @@ async def _initCoreHive(self):
             await self.stormvars.set(s_stormlib_cell.runtime_fixes_key, s_stormlib_cell.getMaxHotFixes())
         self.onfini(self.stormvars)
 
+    async def _initDeprLocks(self):
+        if self.inaugural:
+            locks = (
+                # 2.87.0 - lock out incorrect crypto model
+                ('crypto:currency:transaction:inputs', True),
+                ('crypto:currency:transaction:outputs', True),
```

Comment:
Ahould we go through the model and lock anything set to deprecated instead of using a manually updated list?